stdlib/universe/aggregate_window_test.flux

```diff
@@ -1,40 +1,117 @@
+            {_time: 2019-11-25T00:00:20Z, _measurement: "m0", _field: "f0", t0: "a-0", _value: 1},
+            {_time: 2019-11-25T00:00:30Z, _measurement: "m0", _field: "f0", t0: "a-0", _value: 0},
+            {_time: 2019-11-25T00:00:40Z, _measurement: "m0", _field: "f0", t0: "a-0", _value: 1},
+            {_time: 2019-11-25T00:00:50Z, _measurement: "m0", _field: "f0", t0: "a-0", _value: 1},
+            {_time: 2019-11-25T00:01:00Z, _measurement: "m0", _field: "f0", t0: "a-0", _value: 0},
+            {_time: 2019-11-25T00:00:10Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 1},
+            {_time: 2019-11-25T00:00:20Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 1},
+            {_time: 2019-11-25T00:00:30Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 0},
+            {_time: 2019-11-25T00:00:40Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 1},
+            {_time: 2019-11-25T00:00:50Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 1},
+            {_time: 2019-11-25T00:01:00Z, _measurement: "m0", _field: "f0", t0: "a-1", _value: 0},
+            {_time: 2019-11-25T00:00:10Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 1},
+            {_time: 2019-11-25T00:00:20Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 1},
+            {_time: 2019-11-25T00:00:30Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 0},
+            {_time: 2019-11-25T00:00:40Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 1},
+            {_time: 2019-11-25T00:00:50Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 1},
+            {_time: 2019-11-25T00:01:00Z, _measurement: "m0", _field: "f0", t0: "a-2", _value: 0},
+        ],
+    )
+        |> group(columns: ["_measurement", "_field", "t0"])
+    got = do_test(every: 10s, fn: count)
 
-test _aggregate_window = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: aggregate_window})
+    testing.diff(got, want) |> yield()
+}
+testcase min_with_nulls {
+    want = csv.from(
+        csv: "#datatype,string,long,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,true,true,true,false
+#default,_result,,,,,,
+,result,table,_time,_measurement,_field,t0,_value
+,,0,2019-11-25T00:00:10Z,m0,f0,a-0,1.0
+,,0,2019-11-25T00:00:20Z,m0,f0,a-0,2.0
+,,0,2019-11-25T00:00:30Z,m0,f0,a-0,
+,,0,2019-11-25T00:00:40Z,m0,f0,a-0,3.0
+,,0,2019-11-25T00:00:50Z,m0,f0,a-0,4.0
+,,0,2019-11-25T00:01:00Z,m0,f0,a-0,
+,,1,2019-11-25T00:00:10Z,m0,f0,a-1,1.0
+,,1,2019-11-25T00:00:20Z,m0,f0,a-1,2.0
+,,1,2019-11-25T00:00:30Z,m0,f0,a-1,
+,,1,2019-11-25T00:00:40Z,m0,f0,a-1,3.0
+,,1,2019-11-25T00:00:50Z,m0,f0,a-1,4.0
+,,1,2019-11-25T00:01:00Z,m0,f0,a-1,
+,,2,2019-11-25T00:00:10Z,m0,f0,a-2,1.0
+,,2,2019-11-25T00:00:20Z,m0,f0,a-2,2.0
+,,2,2019-11-25T00:00:30Z,m0,f0,a-2,
+,,2,2019-11-25T00:00:40Z,m0,f0,a-2,3.0
+,,2,2019-11-25T00:00:50Z,m0,f0,a-2,4.0
+,,2,2019-11-25T00:01:00Z,m0,f0,a-2,
```

Comment:
This fill is meant to ensure that empty tables have at least one null row so that we can use `fill()` to fill the values. This is testing the new `table.fill()` and not the one in universe.

Reply:
Ah, I think I get it now. We're inserting null values at specific timestamps where we need a value but don't have one.

Thanks for clarifying!